RecoEgamma/EgammaPhotonProducers/src/ConvertedPhotonProducer.cc

```diff
@@ -537,11 +513,8 @@ void ConvertedPhotonProducer::buildCollections(
             float dCotTheta = -999.;
             reco::TrackRef goodRef;
             std::vector<reco::TransientTrack>::const_iterator iGoodGenTran;
-            for (std::vector<reco::TransientTrack>::const_iterator iTran = t_generalTrk.begin();
-                 iTran != t_generalTrk.end();
-                 ++iTran) {
-              const reco::TrackTransientTrack* ttt =
-                  dynamic_cast<const reco::TrackTransientTrack*>(iTran->basicTransientTrack());
+            for (auto iTran = t_generalTrk.begin(); iTran != t_generalTrk.end(); ++iTran) {
```

Comment:
change to range for here?
`iGoodGenTran`  can be changed to a simple pointer

Reply:
I realized that `iGoodGenTran` was only used once for another variable that was completely unused. Hence it got removed.